### ISSUE TITLE
timing

### ISSUE THREAD

**Issue by lmkirvan** [kirvanai/Gainesville-Quakers-Failed#2]
I got some more info, and it looks like our current provider is going away in April. Is end of March realistic for a first draft?

**Comment by kirvanai** [comment]
Yeah I would say so. I can start trying to put something together this week
if you want! What's the info?

**Comment by lmkirvan** [comment]
Oh the info is just that they aren't going to support past April. 

Here's the existing website again:

http://www.quakercloud.org/cloud/gainesville-quakers

The main thing left to do is look at the options for an easy to host content management system. 

I'll add you to me netlify account that we can use for hosting. 

And here's the email from before, just for posterity. 



Hey Irving,


Here's the existing website:


https://www.quakercloud.org/cloud/gainesville-quakers


Not very exciting. 


There's also a members area that looks like this:


￼


So the members area has something like a CMS embedded in it. This is the modal you see when you go to start a new page--most of these options don't work anymore. 


￼


The design of this is pretty obviously goofy.  It has a news page, an events page, and a calendar that are all basically the same page.  Shrug emoji.  The home page is also just absolutely full of content. way too much stuff! So one of the main things to figure out is what's the useful minimal version of this website. 


The SEYM  (our parent) is a good example of a direction we could go in and at least has a more sensible breakdown of pages.  


https://seymquakers.org/


I looked on https://builtwith.com/seymquakers.org and it was definitely made with wordpress (FWIW). 


Here's a link to the hugo page that discusses CMS options:

https://gohugo.io/tools/front-ends/#prose


There are a couple of options there. 


Of course there are lots of different iterations of static site generators; if you want to get more of a general flavor jamstack.org is a good place to look. 


Hugo is go based, but there are a bunch of javascript based static site generators as well. I've built a site with eleventy before and it was actually super easy and It looks like eleventy works with a few different cms options as well. 


Let's settle on an option and then we can get the repo set up. take a day and let me know what you think makes the most sense. 


I've also archived all the pages you can get to from the existing site here:

https://github.com/lmkirvan/gqm_archive


Let's try to talk again this long weekend. 


Love you brother!


-Lewis 





Sounds good! I'll take a crack at it this weekend and see what I can come up with. Love you too!


-Irving

Show quoted text

**Comment by lmkirvan** [comment]
Please do! Let me know if you want help from me on parts of this.

**Comment by kirvanai** [comment]
So did you want to use Netlify to host? Or was that just more because you
already have an account?

**Comment by lmkirvan** [comment]
Ehh, it's easy. At least for getting it set up, maybe we should use it. It's gonna get like 200 views a week max so cost won't be high even on netlify

**Comment by lmkirvan** [comment]
Hey irv just checking that this is still happening...just checking because I haven't seen any commits

**Comment by kirvanai** [comment]
Yeah sorry I've been moving really slow with this project. I'll admit I've
been a mixture of distracted and also a bit lost in the sauce. I've been
working through a couple demos just trying to get the hang of the framework
and trying to set it up from scratch rather than with a premade binary. I
was really stuck on setting up a compiler for Go using visual studio code
and had to set up Clang which I believe should allow it to run on Mac as
well. I'll go clone that church theme and push it up tonight and see if I
can start filling in the data we need. I'll call you if I am stuck/have any
questions.

**Comment by lmkirvan** [comment]
Hey no problem! We were just talking about it here and I checked the project and didn't see any commits. Please reach out if you get stuck or have questions.

**Comment by kirvanai** [comment]
Just an update for you after a day an a half!  I neglected to read the
quick start guide for Hugo and just restarted the project because I
realized you can't just directly clone the way I could with Angular and it
seemed faster than trying to fix what little I had already done. I found
all the proper guides for starting a project with hugo and using the theme
so I'm making sure to actually follow them now! I'll keep you posted as I
keep working through it though.
